src/pkg/runner/runner.go

```diff
@@ -50,11 +50,17 @@ func Run(tasksFile types.TasksFile, taskName string, setVariables map[string]str
 	// only process includes if the task requires them
 	for _, a := range task.Actions {
 		if strings.Contains(a.TaskReference, ":") {
-			err = runner.importTasks(tasksFile.Includes,config.TaskFileLocation)
-			if err != nil {
-				return err
+			taskReferenceName := strings.Split(a.TaskReference, ":")[0]
+			for _, b:= range tasksFile.Includes{
```

Comment:
What does `b` refer to in this case?

Reply:
not a good sign if you have to ask lol, updated var name to "include"